app.lua

```diff
@@ -67,9 +67,14 @@ require 'responses'
 
 -- Make cookies persistent
 app.cookie_attributes = function(self)
+    local secure = ' Secure;'
+    -- Exclude security restriction in development
+    if package.loaded.config._name == 'development' then
```

Comment:
don't you have this in #105? See feedback there. I don't think we should be hard coding environment names. use a config property.

Reply:
This should be merged before #105. It's just hard to test that without this. ;)